packages/java-edition/src/mcdocAttributes.ts

```diff
@@ -33,6 +33,26 @@ export function registerMcdocAttributes(
 			return ReleaseVersion.cmp(release, config as ReleaseVersion) < 0
 		},
 	})
+	mcdoc.runtime.registerAttribute(meta, 'only', validator.string, {
+		filterElement: (config, ctx) => {
+			if (!config.startsWith('1.')) {
+				ctx.logger.warn(`Invalid mcdoc attribute for "until": ${config}`)
+				return true
+			}
+			return ReleaseVersion.cmp(release, config as ReleaseVersion) == 0
+		},
+	})
+	mcdoc.runtime.registerAttribute(meta, 'except', validator.string, {
+		filterElement: (config, ctx) => {
+			const range = config.split("-")
+			if (range.length != 2 || !range[0].startsWith('1.')|| !range[1].startsWith('1.')) {
```

Comment:
Other than the linting issue the linter points out, there is a missing space before one of the `||`